src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,151 @@
+                Task temp = new Todo(item);
+                tasks.add(temp);
+                taskSum++;
+                System.out.println(Line + "\n" + "     Got it. I've added this task: " + "\n" + "       [T][ ] " + item + "\n" + "     Now you have " + taskSum + " tasks in the list");
+                System.out.println(Line);
+                
+            }
+            else if(command.contains("deadline")){
+                int first = command.indexOf(" ");
+                int check = command.indexOf("deadline");
+                if(check == command.length()-8){
+                    System.out.println(Line);
+                    System.out.println("     ☹ OOPS!!! The description of a deadline cannot be empty.");
+                    System.out.println(Line);
+                    continue;
+                }
+                int itemEnd = command.indexOf("/");
+                String item = command.substring(first,itemEnd);
+                String by = command.substring(itemEnd + 1,command.length());
+                Task temp = new Deadline(item,by);
+                tasks.add(temp);
+                taskSum++;
+                System.out.println(Line + "\n" + "     Got it. I've added this task: " + "\n" + "       [D][ ] " + item + " (" + by + ")" + "\n" + "     Now you have " + taskSum + " tasks in the list");
+                System.out.println(Line);
+            }
+            else if(command.contains("event")){
+                int first = command.indexOf(" ");
+                int check = command.indexOf("event");
+                if(check == command.length()-5){
+                    System.out.println(Line);
+                    System.out.println("     ☹ OOPS!!! The description of a event cannot be empty.");
+                    System.out.println(Line);
+                    continue;
+                }
+                int itemEnd = command.indexOf("/");
+                String item = command.substring(first,itemEnd);
+                String at = command.substring(itemEnd + 1,command.length());
+                Task temp = new Event(item,at);
+                tasks.add(temp);
+                taskSum++;
+                System.out.println(Line + "\n" + "     Got it. I've added this task: " + "\n" + "       [E][ ] " + item + " (" + at + ")" + "\n" + "     Now you have " + taskSum + " tasks in the list");
+                System.out.println(Line);
+            }
+            else{
+                switch (command){
+                    case "list":
+                        printList();
+                        break;
+                    case "bye":
+                        break;
+                    default:
+                        System.out.println(Line);
+                        System.out.println("     ☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                        System.out.println(Line);
+                }
+            }
+        }while(!command.equals("bye"));
+        bye();
+    }
+}
```

Comment:
The logic can be further broken down into simpler function, therefore, consider using more SLAP. 

Can consider using the switch (instead of if-else) statement too to simplify the code.

Function is too bulky and long (more than 30 LOC), consider breaking it down into smaller parts. 

Can consider making the happy path prominent (i.e. make the successful execution path obvious).